internal/builders/gradle/collect_release_artifacts.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+mkdir release-files-for-slsa
+GRADLE_VERSION=$(./gradlew properties -q | grep "version:" | awk '{print $2}')
+
+# Move artifacts from the user-supplied artifact list
+IFS=',' read -ra ARTIFACT_ARRAY <<< "$ARTIFACT_LIST"
+for i in "${ARTIFACT_ARRAY[@]}"
```

Comment:
nit: I know I was the one that suggested this but let's use lowercase for variables local to this file by convention. This makes it easier for the `shellcheck` linter to find issues with code.
```suggestion
IFS=',' read -ra artifact_array <<< "$ARTIFACT_LIST"
for i in "${artifact_array[@]}"
```